mtp_api/apps/core/management/commands/dump_for_ap.py

```diff
@@ -68,25 +68,59 @@ def serialise(self, record):
 
 class CreditSerialiser(Serialiser, serialised_model=Credit):
     headers = Serialiser.headers + [
-        'Internal ID', 'URL',
-        'Date started', 'Date received', 'Date credited',
+        'Internal ID',
+        'URL',
+        'Date started',
+        'Date received',
+        'Date credited',
         'Amount',
-        'Prisoner number', 'Prisoner name', 'Prison',
-        'Sender name', 'Payment method',
-        'Bank transfer sort code', 'Bank transfer account', 'Bank transfer roll number',
-        'Debit card number', 'Debit card expiry', 'Debit card billing address',
-        'Sender email', 'Sender IP address',
+        'Prisoner number',
+        'Prisoner name',
+        'Prison',
+        'Owner username',
+        'Blocked',
+        'Sender name',
+        'Payment method',
+        'Bank transfer sort code',
+        'Bank transfer account',
+        'Bank transfer roll number',
+        'Debit card first six digits',
+        'Debit card last four digits',
+        'Debit card expiry',
+        'Debit card billing address line 1',
+        'Debit card billing address line 2',
+        'Debit card billing address city',
+        'Debit card billing address postcode',
+        'Debit card billing address country',
+        'Sender email',
+        'Sender IP address',
         'Status',
         'NOMIS transaction',
         'WorldPay order code',
+        'Security check status',
+        'Security check codes',
+        'Exported at',
     ]
 
+    exported_at_local_time = timezone.now()
```

Comment:
I think that was my intention, so that all the records have exactly the same date. That's what Joe implied the data team were expecting, hence putting it at class level. Do you think that a date with milliseconds difference per row is more valuable?

Reply:
Hopefully the recent change init methods clean that up - although correct me if I'm potentially breaking anything by not calling super() in the init method, but don't think that will override the BaseCommand init where it is, but was not 100% sure.

Reply:
@ushkarev This has changed again - will setting the `exported_at` time in `__init_subclass__` as per the change in 2c45ea6?

Reply:
i think we need it in an `__init__` method because `__init_subclass__` is called at [subclass creation](https://docs.python.org/3/reference/datamodel.html#customizing-class-creation)

Reply:
@ushkarev Ah right, I thought that `__init_subclass__` might be called at instantiation, but if I understand that  correctly, date would not be set at run time on line 38 unless we include `__init__` on the Serialiser class itself?

Reply:
Anyway, how does that look, and if it's OK, I might resolve the merge conflicts on a new branch and get all the branches ready to merge to master